packages/examples/cvat/exchange-oracle/src/endpoints/middleware.py

```diff
@@ -44,8 +45,7 @@ async def profile_request(request: Request, call_next: Callable):
         response_data = profiler.output(renderer=renderer)
         if profile_type == "html":
             return HTMLResponse(response_data)
-        else:
-            return StreamingResponse(iter(response_data))
+        return StreamingResponse(iter(response_data))
```

Comment:
There are several similar places where the same comment would fit. I'd say that while correct, the early return pattern is typically used either for an optimization or for secondary use-cases. In this case both branches have equal priority, so I'd prefer using just a normal if-else.

Reply:
I have no objections against someone writing their code this way, but I'm against enforcing it via automatic formatting.

Reply:
Although, the point was exactly in enforcing one consistent style throughout the project and, so there would not be multiple ways of doing the same thing, I've disabled the rule and reverted the changes for now so we can merge this.